moon/powerups/server/viper.moon

```diff
@@ -0,0 +1,70 @@
+include "base.lua"
+
+POWERUP_ID = "viper-powerup"
```

Comment:
You might think this a bit much, but I think `POWERUP_ID` should come from `PowerupManager.register(powerupId)` or something, just so that we can _guarantee_ there aren't any conflicts:
```moon
register: (pid) =>
  counter = 1
  registered_id = pid
  while CFCPowerups[registered_id]
     registered_id = pid .. '-' .. tostring(counter++)

  CFCPowerups[registered_id] = true
  registered-id
```

or w/e, idk